### ISSUE TITLE
Comments on Projectile Motion, Part 3

### ISSUE THREAD

**Issue by JacquesCarette** [smiths/caseStudies#245]
Here are, at a higher level, the things that would be nice to also make explicit:
- what are the learning objectives of this lesson?
- what are the assumptions?
- what are the things that are somehow considered 'inputs', and what are those unknown/output/desired?
- what is the relation between the model and the example(s) ? There are several refinement steps going on here, and they are all very implicit.
- what is the purpose of those print statement? [This is more a meta-question, since I do know what their purpose is, but it reads funny.]
- aren't there ways to make Out cells in notebooks?

**Comment by smiths** [comment]
This issue is related to #247.

**Comment by smiths** [comment]
Commit 1a7b1c7, together with previous commits closes this issue.

Learning objectives and explicit assumptions have been added.  The inputs are separated into knowns and unknowns.  The steps for solving a projectile motion problem are made explicit and linked throughout the example.  I've also removed any print statements that weren't adding new information.

With respect to outputs, they can be displayed on their own (as is done for the answers), but they cannot be incorporated (as far as I can tell) into the markdown text documentation.
